spec/requests/users_spec.rb

```diff
@@ -0,0 +1,13 @@
+RSpec.describe 'Users', type: :request do
+  describe 'GET /index' do
+    it 'returns http success' do
+      get '/users'
+      expect(response).to have_http_status(:success)
+    end
+    it 'assigns @users' do
```

Comment:
```suggestion

    it 'assigns @users' do
```

Espaço entre declarações para facilitar a leitura